protocols/kad/src/behaviour.rs

```diff
@@ -441,6 +441,11 @@ where
         &mut self.store
     }
 
+    /// Returns the number of iterative queries that are currently being performed.
+    pub fn num_active_iterative_queries(&self) -> usize {
```

Comment:
```suggestion
    pub fn num_queries(&self) -> usize {
```
Because there is no such thing as an inactive query, I don't think `active` is a useful distinction and may be misleading.